api/api.yaml

```diff
@@ -151,8 +158,10 @@ paths:
         - name: projectIdOrName
           in: path
           required: true
+          description: The unique identifier or name of the project. 
```

Comment:
can you revise all descriptions of "projectIdOrName" to be "The Ampersand project ID or project name"